CHANGES.rst

```diff
@@ -29,6 +29,9 @@ Bug fixes
 
   * Fix warning using `EXT_SUFFIX` config variable instead of deprecated `SO` variable. See :issue:`381`.
 
+* Honor the `MACOSX_DEPLOYMENT_TARGET` environment variable if it is defined on
+  macOS. See :issue`441`.
```

Comment:
```
[...] Thanks :user:`certik` for the contribution. See :issue`441`.
```

Reply:
Fixed in eec72d4815fd6d9fd8ce56f9ca196379a482165d.